quest/util/config.py

```diff
@@ -1,8 +1,7 @@
 """Module wide settings.
 
 """
-
-import appdirs
+from os.path import expanduser
```

Comment:
Rather than specifically importing expand user I would suggest just using the full dot-path (i.e. `os.path.expanduser`) when you need to use this function. (We are already import `os`).